packages/event-migration/src/index.ts

```diff
@@ -0,0 +1,199 @@
+    TARGET_DB_USE_SSL: z
+      .enum(["true", "false"])
+      .transform((value) => value === "true"),
+  })
+  .transform((c) => ({
+    sourceDbUsername: c.SOURCE_DB_USERNAME,
+    sourceDbPassword: c.SOURCE_DB_PASSWORD,
+    sourceDbHost: c.SOURCE_DB_HOST,
+    sourceDbPort: c.SOURCE_DB_PORT,
+    sourceDbName: c.SOURCE_DB_NAME,
+    sourceDbSchema: c.SOURCE_DB_SCHEMA,
+    sourceDbUseSSL: c.SOURCE_DB_USE_SSL,
+    targetDbUsername: c.TARGET_DB_USERNAME,
+    targetDbPassword: c.TARGET_DB_PASSWORD,
+    targetDbHost: c.TARGET_DB_HOST,
+    targetDbPort: c.TARGET_DB_PORT,
+    targetDbName: c.TARGET_DB_NAME,
+    targetDbSchema: c.TARGET_DB_SCHEMA,
+    targetDbUseSSL: c.TARGET_DB_USE_SSL,
+  }));
+export type Config = z.infer<typeof Config>;
+
+export const config: Config = {
+  ...Config.parse(process.env),
+};
+
+export type DB = IDatabase<unknown>;
+
+export function initDB({
+  username,
+  password,
+  host,
+  port,
+  database,
+  schema,
+  useSSL,
+}: {
+  username: string;
+  password: string;
+  host: string;
+  port: number;
+  database: string;
+  schema: string;
+  useSSL: boolean;
+}): DB {
+  const pgp = pgPromise({
+    schema,
+  });
+
+  const conData = new ConnectionString(
+    `postgresql://${username}:${password}@${host}:${port}/${database}`
+  );
+
+  const dbConfig: IConnectionParameters<IClient> = {
+    database: conData.path !== undefined ? conData.path[0] : "",
+    host: conData.hostname,
+    password: conData.password,
+    port: conData.port,
+    user: conData.user,
+    ssl: useSSL ? { rejectUnauthorized: false } : undefined,
+  };
+
+  return pgp(dbConfig);
+}
+console.log("Starting migration");
+
+console.log("Initializing connections to source database");
+const sourceConnection = initDB({
+  username: config.sourceDbUsername,
+  password: config.sourceDbPassword,
+  host: config.sourceDbHost,
+  port: config.sourceDbPort,
+  database: config.sourceDbName,
+  schema: config.sourceDbSchema,
+  useSSL: config.sourceDbUseSSL,
+});
+
+console.log("Initializing connections to target database");
+const targetConnection = initDB({
+  username: config.targetDbUsername,
+  password: config.targetDbPassword,
+  host: config.targetDbHost,
+  port: config.targetDbPort,
+  database: config.targetDbName,
+  schema: config.targetDbSchema,
+  useSSL: config.targetDbUseSSL,
+});
+
+console.log("reading events from source database");
+const originalEvents = await sourceConnection.many(
+  "SELECT event_ser_manifest, event_payload, write_timestamp FROM event_journal order by ordering ASC"
+);
+
+const idVersionHashMap = new Map<string, number>();
+
+for (const event of originalEvents) {
+  console.log(event);
+  const { event_ser_manifest, event_payload, write_timestamp } = event;
+
+  const eventType = match(
+    event_ser_manifest
+      .replace("it.pagopa.interop.catalogmanagement.model.persistence.", "")
+      .split("|")[0]
+  )
+    .with("CatalogItemDescriptorItemAdded", () => "EServiceDescriptorAdded")
```

Comment:
Since we have this exception, that looks like the only one, we can fix the name of that event before production starts? So that they all adhere to the same criteria. We already did some events fixing in this other PR for example: https://github.com/pagopa/interop-be-monorepo/pull/253/files

Reply:
I double checked and in reality, the original event is already stored as `...CatalogItemDescriptorAdded`, it was an error present in the excel with the information related to the migration from events V1 to events V2